### DISCUSSION TITLE
Inconsistent positioning of the quick bar when the horizontal scrollbar is used in browsers

### DISCUSSION THREAD

**Discussion by BjornA77** [Security-Onion-Solutions/securityonion#1811] [General] [answered]
The quick bar seems to be affected when the horizontal scrollbar is used in the browser window. I have tested with both Chrome and Firefox in Windows and with Chromium in Centos and the results are the same.

If you run the browser in a size so that the horizontal scrollbar shows this happen. I took some screenshots when clicking on:

rule.name
rule.category
event.severity_label
log.id.uid
network.community_id

It moves more and more to the right and when clicking on network.community_id the quick bar can no longer be seen so it's not possible to use it. The problem start when all fields can not fit horizontally in the browser window and the scrollbar is activated automatically by the browser. It does not help to move the slider max to the right either.

Since the horizontal scrollbar is activated quite often if you use a smaller browser window, in VM:s and so on depending on resolution and scaling it would be great if this could be resolved.

![image](https://user-images.githubusercontent.com/73762058/98052019-6c0fb200-1e35-11eb-9f8d-eee70487633d.png)
![image](https://user-images.githubusercontent.com/73762058/98052055-7cc02800-1e35-11eb-8f3e-6826dccca431.png)
![image](https://user-images.githubusercontent.com/73762058/98052065-85186300-1e35-11eb-913e-3755336c5028.png)
![image](https://user-images.githubusercontent.com/73762058/98052085-8e093480-1e35-11eb-8d9b-ceff303a9273.png)
![image](https://user-images.githubusercontent.com/73762058/98052099-97929c80-1e35-11eb-9df8-b75646cc59da.png)

**Comment by dougburks** [comment, marked as answer]
We've completely redesigned the Quick Action Bar including https://github.com/Security-Onion-Solutions/securityonion/issues/1644 and https://github.com/Security-Onion-Solutions/securityonion/issues/1570.

Look for these changes in version 2.3.10! :smile:
